debug/gdbserver.py

```diff
@@ -1015,6 +1015,32 @@ def test(self):
             if privilege in self.supported:
                 assertEqual(actual, privilege)
 
+class ComplianceTest(GdbTest):
+
+    def test(self):
+        output = self.gdb.command("monitor riscv test_compliance")
+        #TODO: Fix OpenOCD to remove the -rtos riscv requirement
+        assert(("ALL TESTS PASSED") in output or ("Please run with -rtos riscv to run compliance test.") in output)
+
+    def compile(self):
+        pass
+
+class SystemBusAccessConfigRegTest(GdbTest):
+
+    def test(self):
+        result = self.gdb.command("monitor riscv dmi_read 0x38")
+        if (int(result, base=16) == 0):
+            return 'not_applicable'
+        
+        self.gdb.command("monitor riscv set_prefer_sba on")
+        command = "monitor riscv test_sba_config_reg 0x%08x %d 0x%08x off" % (
```

Comment:
When I run this test I get:
```
(gdb) monitor riscv test_sba_config_reg 0x1212340000 100 0x121233fffc off
invalid subcommand "test_sba_config_reg 0x1212340000 100 0x121233fffc off"
```

Reply:
and you're using latest OpenOCD?

Reply:
No. I had the latest source, but hadn't rebuilt. Now the command starts correctly.